resources/com/griddynamics/devops/mpl/modules/Build/GradleBuild.groovy

```diff
@@ -0,0 +1,5 @@
+/**
+ * Simple Gradle Build
+ */
+
+sh './gradlew clean build'
```

Comment:
Hmmm, not sure it's a great jenkins-way - maybe use tool of jenkins? Just imagine the strict environment where is no access to the internet (quite common in the most companies).